src/util.js

```diff
@@ -0,0 +1,19 @@
+/**
+   Public util functions for the SDK.
+ */
+
+/**
+   Constructs an image variant definition query param. Does not validate what
+   param names or values, as that is left for the API.
+*/
+// eslint-disable-next-line import/prefer-default-export
+export const imageVariantURLStr = opts => {
+  if (!opts || typeof opts !== 'object') {
+    throw new Error('Image variant options parameter must an Object.');
+  }
+
+  return Object.entries(opts)
+    .map(([k, v]) => (v ? `${k}:${v}` : null))
+    .filter(entry => !!entry)
+    .join(';');
+};
```

Comment:
I don't see anything here that is specific to image variants. Only that we use this format only for those at this point, right? So this is a very generic object -> URL param serialization.

Reply:
Like mentioned in the PR description, this would be another approach now that the function has no validation for params. My main concern with the more generic naming was that, even though we have semi-formally declared the core query param map format, this would mean that the fn should also support arrays for extended data filtering queries, which I didn't want to add in the scope of this project. Also, I'm thinking if we want to "lock" the query parameter map syntax like that. Then again, I'm starting to lean more towards the more generic naming, as it feels a bit backwards to make this image variant specific and the possibly deprecate this later just to declare the same functionality in a more generic form.

Reply:
The method is now renamed with a more generic use case in mind.